tests/ethernet/eth.py

```diff
@@ -0,0 +1,23 @@
+import utils
+
+
+def captures_ok(api, cfg, size, src, dst):
+    """
+    Returns normally if patterns in captured packets are as expected.
+    """
+    cap_dict = utils.get_all_captures(api, cfg)
+    src_list = []
+    dst_list = []
+    length = []
+    for k in cap_dict:
+        for b in cap_dict[k]:
+            dst_list.append(
+                ":".join(['{:02X}'.format(byt) for byt in b[0:6]])
```

Comment:
wouldn't it make sense to move it to `mac_addr_from_bytes(b[0:6])` ? If we haven something similar for IPv4 as well, we should consider the same for it as well.

Reply:
As this was added at protocol level, did not feel to add as definition.
will move to definition